resources/sdk/clisdkclient/extensions/cmd/usage/usagequery_test.go

```diff
@@ -0,0 +1,46 @@
+package usage
```

Comment:
Testing against swagger definition here. 
Currently only using path (including path params), method and description. 
I'll add tests for query params etc if they're ever required.